src/EditorFeatures/Core/BlockCommentEditing/BlockCommentEditingOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.BlockCommentEditing
+{
+    internal class BlockCommentEditingOptions
+    {
+        public static readonly PerLanguageOption2<bool> AutoInsertBlockCommentStartString = new("csharp_automatically_insert_block_comment_start_string", defaultValue: true);
```

Comment:
>csharp_automatically_insert_block_comment_start_string [](http://example.com/codeflow?start=97&length=54)

Can we move this to C# specific project?

Reply:
Moved